src/Libraries/RevitNodes/Properties/Resources.en-US.resx

```diff
@@ -360,7 +360,10 @@
   <data name="TypeNotFound" xml:space="preserve">
     <value>Type not found in document.</value>
   </data>
-  <data name="ViewDoesNotSupportAnnotations" xml:space="preserve">
-    <value>View does not support annotations.</value>
+  <data name="CategoryArgumentException" xml:space="preserve">
```

Comment:
Nice! By adding these here, your strings will now be localized into every language that Revit supports :)